### PR TITLE
ci: 9262 gha build

### PR DESCRIPTION
**Pull request by Mikemosca**
### Summary

Add build and test, remove appveyor

### CODE REVIEW

**Comment by mediumTaj** [comment]
Can we do this in Windows as well? Maybe a separate story

**Comment by Mikemosca** [comment]
> Can we do this in Windows as well? Maybe a separate story

I think so, We might need to create a separate build.yml as I do not think it is as easy to just add windows as a os.  I will create an issue for this.

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  workflow_dispatch:
+
+jobs:
+  build_test:
+    name: Build and Test on dotnet ${{ matrix.dotnet-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        dotnet-version: ['2.1.x']
+        os: [ubuntu-latest]
```
Comment (mediumTaj) [inline .github/workflows/build-test.yml:24]: Should we be doing latest here? should we lock to a particular version?

**Review by mediumTaj — APPROVED** [review]

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (Mikemosca) [reply .github/workflows/build-test.yml:24]: I tried it didn't work.  I think we would have to change some of the .csproj files.  I will create another issue to look into this.

**Pull request merged**